pkg/controller/operators/catalog/operator.go

```diff
@@ -1056,6 +1067,25 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 				// Attempt to create the CRD.
 				_, err = o.OpClient.ApiextensionsV1beta1Interface().ApiextensionsV1beta1().CustomResourceDefinitions().Create(&crd)
 				if k8serrors.IsAlreadyExists(err) {
+					currentCRD, _ := o.OpClient.ApiextensionsV1beta1Interface().ApiextensionsV1beta1().CustomResourceDefinitions().Get(crd.GetName(), metav1.GetOptions{})
+					// Compare 2 CRDs to see if it needs to be updatetd
+					if !reflect.DeepEqual(crd, *currentCRD) {
+						// Verify CRD ownership, only attempt to update if
+						// CRD has only one owner
+						// Example: provided=database.coreos.com/v1alpha1/EtcdCluster
+						apiKey := fmt.Sprintf("provided=%s/%s", crd.APIVersion, crd.Kind)
```

Comment:
This is not the correct format for the key (look at what was chosen in api.go). The example text should look like:
provided=packages.operators.coreos.com/v1/PackageManifest provided=cluster.com/v1alpha1/ins-mcvbv...
At first I thought it might just be easier to change the key format to use the CRD name without splitting it, but maybe for consistency with the api services it should stay as is.